lib/include/jxl/codestream_header.h

```diff
@@ -257,7 +262,7 @@ typedef struct {
   /** Whether alpha channel uses premultiplied alpha. Only applicable if
    * type is JXL_CHANNEL_ALPHA.
    */
-  JXL_BOOL alpha_associated;
+  JXL_BOOL alpha_premultiplied;
```

Comment:
Mention this change in the changelog in a new Unreleased.Changed section since this will go in 0.6

Reply:
I'll do that in a next merge request soon since this involves chapter changes in the changelog during the 0.5/0.6 transition